configs/webpack-config-compass/src/loaders.ts

```diff
@@ -41,6 +41,13 @@ export const javascriptLoader = (args: ConfigArgs) => ({
           require.resolve('@babel/plugin-proposal-decorators'),
           { legacy: true },
         ],
+        args.coverage && [
```

Comment:
I think having it here makes sense, what feels a bit out of place is the `cwd` option as we probably only want it applied like that for `packages/compass` not for all packages that use this configuration. Maybe have a separate argument added for that and passed to the config in `packages/compass/webpack.config.js` instead of always set to the root of the monorepo? Don't have a strong preference for that though

Reply:
What if I make args.coverage an optional string and if it is set we set it to the path we want to generate coverage for?

Reply:
Sounds good too, yeah